### DISCUSSION TITLE
Github Copilot on multiple devices?

### DISCUSSION THREAD

**Discussion by muhammadaman97** [community/community#26702] [Actions]
<p>Can I use github copilot on multiple devices?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there! Visit the <a href="https://github.com/github/feedback/discussions/categories/copilot-feedback">Feedback forum</a> to ask questions, share bugs or feedback, or chat with other users in the Preview. The GitHub Copilot team will respond as often as possible, but we also welcome you to share your experiences and help others in the community.</p>
